### PR TITLE
Describe Plex "scan_clients" button

### PR DESCRIPTION
**Pull request by jjlawren**
## Proposed change

Adds a description of the "scan clients" `button` entity added to Plex.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/67055
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by jjlawren** [comment]
Adds missing documentation for an existing feature.

File: source/_integrations/plex.markdown [hunk 1]
```diff
@@ -97,6 +98,39 @@ The library sensors are disabled by default, but can be enabled via the Plex int
...
+
+```yaml
+play_plex_on_tv:
+  sequence:
+    - service: media_player.select_source
+      target:
+        entity_id: media_player.smart_tv
+      data:
+        source: "Plex"
+    - wait_template: "{{ is_state('media_player.smart_tv', 'On') }}"
```
Comment (frenck) [inline source/_integrations/plex.markdown]:
```suggestion
    - wait_template: "{{ is_state('media_player.smart_tv', 'on') }}"
```

Comment (frenck) [reply source/_integrations/plex.markdown]: Actually, this should be replaced with a wait-for trigger without templating.

File: source/_integrations/plex.markdown [hunk 2]
```diff
@@ -97,6 +98,39 @@ The library sensors are disabled by default, but can be enabled via the Plex int
...
+      timeout: "00:00:10"
+    - service: button.press
+      target:
+        entity_id: button.scan_clients_plex
+    - wait_template: "{{ not is_state('media_player.plex_smart_tv', 'unavailable') }}"
```
Comment (frenck) [inline source/_integrations/plex.markdown:126]: Same as above

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/plex.markdown (hunk 1 above)
Comment (jjlawren) [reply source/_integrations/plex.markdown]: Good call, I think this was originally added before that was available. 👍

Comment (jjlawren) [reply source/_integrations/plex.markdown]: Hmm, I just updated this but now I'm reconsidering. A `wait_for_trigger` will not fire if the state was already correct and the timeout will impose an unnecessary delay.

Comment (frenck) [reply source/_integrations/plex.markdown]: Right... would be possible to wrap it with an `if` statement, however, that is not available in the current release (added in 2022.5).

**Review by frenck — APPROVED** [review]
Thanks, @jjlawren 👍

**Pull request merged**